react-native-scripts/template/README.md

```diff
@@ -142,6 +143,34 @@ You can optionally use a [plugin for your IDE or editor](https://flow.org/en/doc
 
 To learn more about Flow, check out [its documentation](https://flow.org/).
 
+## Adding ESLint and Prettier
+
+[ESLint](http://eslint.org/) is a configurable JavaScript linter. ESLint can help standardize a style guide across the team.
+
+To add eslint to a Create React Native App project, follow these steps:
```

Comment:
eslint should be ESLint here -- make sure to use casing consistently when using outside of context of executing command :) I might have missed somewhere else